app/_src/policies/meshmetric.md

```diff
@@ -30,13 +30,33 @@ If you haven't already read the [observability docs](/docs/{{ page.version }}/ex
 
 ## TargetRef support matrix
 
+{% if_version gte:2.6.x %}
+{% tabs targetRef useUrlFragment=false %}
+{% tab targetRef Sidecar %}
+| `targetRef`             | Allowed kinds                                            |
+| ----------------------- | -------------------------------------------------------- |
+| `targetRef.kind`        | `Mesh`, `MeshSubset`, `MeshService`, `MeshServiceSubset` |
+{% endtab %}
+
+{% tab targetRef Builtin Gateway %}
+| `targetRef`             | Allowed kinds                                            |
+| ----------------------- | -------------------------------------------------------- |
+| `targetRef.kind`        | `Mesh`                                                   |
```

Comment:
not sure here for the gateway, FYI @Automaat 

validation def won't allow it https://github.com/kumahq/kuma/blob/df79d3a77cae0d72fddd4a7bc87857ab4db8c486/pkg/plugins/policies/meshmetric/api/v1alpha1/validator.go#L24

Reply:
changed, now we support `MeshGateway` targetRef